docs/import/add_disaster.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Add New Disaster</title>
+  <!-- TODO(janakr): Switch back to compiled version in production. -->
+  <!-- TODO(janakr): This is pinned to a particular version, but we should be targeting the live
+       version, whatever it is. -->
+  <script src="https://rawcdn.githack.com/google/earthengine-api/3bb86bfc4f3d9eed98220f3d225b414982915b86/javascript/build/ee_api_js_debug.js"></script>
+  <!-- TODO(janakr): download this script to lib. -->
+  <script src="https://apis.google.com/js/api.js"></script>
+
+  <script src="https://www.gstatic.com/firebasejs/6.3.3/firebase-app.js"></script>
+  <script src="https://www.gstatic.com/firebasejs/6.3.3/firebase-firestore.js"></script>
+  <script src="https://www.gstatic.com/firebasejs/7.2.1/firebase-auth.js"></script>
+  <!-- Load jQuery, a 3rd-party library. -->
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+  <script src="https://apis.google.com/js/api.js"></script>
+  <script type="module" src="./add_disaster_startup.js"></script>
+</head>
+<body>
+
+<form action="" id="disaster-picker">
+  <label for="disaster">Disaster: </label>
+  <select id="disaster" style="width: 160px"></select>
+  <span> </span>
+  <button type="button" id="create-new-disaster">Create New Disaster</button>
+</form><p></p>
+<div id="status"></div>
+<div id="new-disaster" hidden="true">
+  <form action="" id="new-disaster-form">
+    <label for="name">Disaster Name: </label><input id="name"><label for="name"><i> ex: harvey</i></label><br>
+    <label for="year">Disaster Year: </label><input id="year"><label for="year"><i> ex: 2017</i></label><br>
+    <!-- TODO(#207): support us territory data sets and add them here -->
+    <label for="states">State(s): </label><select multiple="multiple" id="states">
```

Comment:
This picker is pretty small. Maybe make it a bit bigger by default?

Reply:
I meant higher, not wider :)

Reply:
Ahh, done.